.circleci/config.yml

```diff
@@ -46,9 +46,73 @@ jobs:
       - run:
           command: |
             ~/SmartPeak2/.circleci/run_examples.sh ~/SmartPeak2_build/bin
+      - run:
+          command: |
+            set -o xtrace
+            cd ~/SmartPeak2_build
+            cpack -V
+            ls -lh ~/SmartPeak2_build | grep -a .deb
+            dpkg -c ~/SmartPeak2_build/*.deb
+            mkdir -p /tmp/packages
+            cp ~/SmartPeak2_build/*.deb /tmp/packages
```

Comment:
So cool!

Reply:
Thanks for the feedback, I have removed the unused/unrelated files pertaining to the packaging process. It's pass the CI pipelines on both platforms.

For Windows CPack, I will have to perform this somewhere locally while inspecting why AppVeyor is failing with OpenMS build.